CentOS: Incorrect trusted_file path specified for libstdbuf in manifest.template for LTP

## Description of the problem
Trusted file path of libstd should not be dependent on arch_libdir, it should be solely dependent upon coreutils_libdir

In Manifest.template

> loader.env.LD_PRELOAD = "{{ coreutils_libdir }}/libstdbuf.so"
>   translates in CentOS:
>   ```loader.env.LD_PRELOAD = "/usr/libexec/coreutils/libstdbuf.so"```
>   translates in Ubuntu:
>   `loader.env.LD_PRELOAD = "/usr/lib64/arch_libdir/coreutils/libstdbuf.so"`
> 
> sgx.trusted_files.libstdbuf = "file:/usr/{{ arch_libdir }}/coreutils/libstdbuf.so"
>   translates in CentOS:
>   ```loader.env.LD_PRELOAD = "/usr/lib64/arch_libdir/coreutils/libstdbuf.so"```
>   translates in Ubuntu:
>   `loader.env.LD_PRELOAD = "/usr/lib64/arch_libdir/coreutils/libstdbuf.so"`
> 

## Steps to reproduce
cd LibOS/shim/test/ltp
make SGX=1 all sgx-tokens

## Expected results
SGX-Tokens should be able to generate successfully.

## Actual results
make[1]: *** No rule to make target '/usr/lib64/coreutils/libstdbuf.so', needed by 'abort01.manifest.sgx'.  Stop.
make[1]: *** Waiting for unfinished jobs....
printf "passwd: compat\ngroup: compat\nshadow: compat\nhosts: files\n" > etc/nsswitch.conf
printf "root:x:0:0:root:/root:/bin/bash\nnobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin\n" > etc/passwd
make[1]: Leaving directory '/home/intel1/Anjali/graphene/LibOS/shim/test/ltp/install-sgx/testcases/bin'
make: *** [Makefile.LTP:58: build-manifest] Error 2

## Solution
`sgx.trusted_files.libstdbuf = "file:{{ coreutils_libdir }}/libstdbuf.so"`
<!-- ## Additional information -->
<!-- if applicable, uncomment and fill this section -->

--- a/LibOS/shim/test/ltp/manifest.template
+++ b/LibOS/shim/test/ltp/manifest.template
@@ -39,6 +39,6 @@ sgx.trusted_files.libdl = "file:{{ graphene.runtimedir() }}/libdl.so.2"
 sgx.trusted_files.libm = "file:{{ graphene.runtimedir() }}/libm.so.6"
 sgx.trusted_files.libpthread = "file:{{ graphene.runtimedir() }}/libpthread.so.0"
 sgx.trusted_files.librt = "file:{{ graphene.runtimedir() }}/librt.so.1"
-sgx.trusted_files.libstdbuf = "file:/usr/{{ arch_libdir }}/coreutils/libstdbuf.so"
+sgx.trusted_files.libstdbuf = "file:{{ coreutils_libdir }}/libstdbuf.so"
 
 sgx.allowed_files.tmp = "file:/tmp"

